pages/common/projucer.md

```diff
@@ -0,0 +1,28 @@
+# projucer
```

Comment:
```suggestion
# Projucer
```

Reply:
Just curious, Will there be linter errors for having Uppercase in the heading?

Reply:
There won't.

<img width="876" alt="a screenshot showing that it works fine" src="https://user-images.githubusercontent.com/35269695/193413285-4e683275-0b59-49a1-a5b7-270060d67ce9.png">

Reply:
> There won't.
> 
> <img width="876" alt="a screenshot showing that it works fine" src="https://user-images.githubusercontent.com/35269695/193413285-4e683275-0b59-49a1-a5b7-270060d67ce9.png">
> 

Thanks 👍🏻